config/routes.rb

```diff
@@ -85,11 +85,23 @@
   end
 
   if TeSS::Config.feature['collections'] == true
-    resources :collections, concerns: [:collaboratable, :activities]
+    resources :collections, concerns: %i[collaboratable activities] do
+      # in the future it could be considered to expand this to multiple collections
+      # at the same view, in a kind of matrix-view, useful for people who manage several
+      # collections.
+      member do
+        %w[events materials].each do |item|
+          return unless TeSS::Config.feature[item]
```

Comment:
this return may cause trouble, should be `next`. I notice we don't have rubocop or anything like that on this repo, should we set that up?

Reply:
and generally I don't really like disabling features at the routes level, IMO it should be via a `before_action` in the controllers. I'll probably create an issue relating to this.